frontend/src/app/main/data/workspace.cljs

```diff
@@ -412,6 +421,28 @@
         (->> (rp/mutation :rename-file params)
              (rx/ignore))))))
 
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Local storage Flags Manipulation
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(def storeable-layout-flags #{:colorpalette})
+(def storeable-workspace-local-flags #{:selected-palette :selected-palette-colorpicker})
+
+(defn store-layout-flags!
+  [state flags]
+  (doseq [item (filter #(contains? storeable-layout-flags %) flags)]
```

Comment:
There is a clojure idiom that we use a lot: when you call a set as if it were a function, it returns the equivalent to using `contains?` on it. So you can replace this line with this one:
```clojure
(doseq [item (filter storeable-layout-flags flags)]
```

Reply:
Oooh, cool

Reply:
done!